MIGRATING.md

```diff
@@ -105,13 +106,14 @@ Both:
   `Coin::new` and returns one `Coin`.
 - Remove the 4th argument (contract balance) from all calls to `mock_env`, this
   is no longer stored in the environment.
-- `mock_dependencies` and `mock_instance` take a 2nd argument to set the
-  contract balance (visible for the querier). If you need to set more balances,
-  use `mock_XX_with_balances`. The follow code block explains:
+- `dependencies` was renamed to `mock_dependencies`. `mock_dependencies` and
+  `mock_instance` take a 2nd argument to set the contract balance (visible for
+  the querier). If you need to set more balances, use `mock_XX_with_balances`.
+  The follow code block explains:
 
   ```rust
   // before: balance as last arg in mock_env
-  let mut deps = mock_dependencies(20);
+  let mut deps = dependencies(20);
```

Comment:
Huh? I thought this was `mock_dependencies`. The line above says so as well.

Reply:
This block is titles with "before", and in 0.7 it was `dependencies`

Reply:
Foolish me, just looking at the diff, not the context.

Sorry for the false alarm. Looks good to merge.